### ISSUE TITLE
Ignore parameter not working?

### ISSUE THREAD

**Issue by sivertjoe** [dermesser/memoize#33]
Tried using the example 
```rs
#[memoize(Ignore: count_calls)]
fn add(a: u32, b: u32, count_calls: &mut u32) -> u32 {
    // Keep track of the number of times the underlying function is called.
	*count_calls += 1;
	a + b
}
```
But it gives me the error
```
: custom attribute panicked
 --> main.rs:4:1
  |
4 | #[memoize(a: u32, b: u32, count_calls: &mut u32)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: called `Result::unwrap()` on an `Err` value: Error("expected one of: `Capacity`, `TimeToLive`, `SharedCache`, `CustomHasher`, `HasherInit`")
```

**Comment by u9g** [comment]
It's not released on master yet, you can change your cargo.toml to have `memoize = {git = "https://github.com/dermesser/memoize"}` if you would like to use the version with the ignore parameter.

**Comment by dermesser** [comment]
I've released `Ignore` with 0.4.2, you can update `memoize` now.
